app/src/main/java/com/example/voyagerx/repository/model/User.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.voyagerx.repository.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class User (
+    @PrimaryKey(autoGenerate = true)
+    val id: Int,
+
+    val email: String,
+    val password: String,
+
+    var name: String,
+    var location: String,
+    var bio: String,
+
+    var favoriteLaunches: List<Launch>
```

Comment:
I think we should make these nullable as well since the user won't be filling these fields out when signing up for an account. We will still need to create the user, but we won't have any info to populate these fields.

Reply:
Good point. These can either be nullable or given default values as seen here: https://kotlinlang.org/docs/functions.html#default-arguments. That way we can minimize the need for null checks throughout our code.